HISTORY.rst

```diff
@@ -5,6 +5,11 @@ History
 Unreleased
 ----------
 
+1.9.5 (2019-10-18)
```

Comment:
Leave this line out, we should add this once we actually release a version, for now stuff should be kept under the unreleased label

Reply:
True, but there are now more changes (so requires a rebase anyway) plus the date is not correct now.

Reply:
Makes sense, I will just move that to `Unreleased`.